data/index.php

```diff
@@ -279,7 +289,15 @@ function selectSet(id) {
 				</select><br/>
 				<input type="text" name="ids" id="idlist" placeholder="2,5,19-23" style="width:330px; margin-top:5px;"/>
 			</fieldset>
-			
+			<fieldset id="limit">
+				<legend>Limit results (0=all)</legend>
```

Comment:
Maybe say "Limit number of results (leave empty to see all)" instead (and also remove the placeholder below)?

Reply:
Yes, I considered using (Optional) as well.
Also I am thinking that if the value is not numeric regex against \d+,\s?\d+ to see if the user has specified start and count for the limit.

Reply:
Allowing limit,offset would indeed be fine as well. I believe Harmen is planning to add some additional documentation about the supported parameters to the box to the right, so maybe it's ok to support, but not mention the offset parameter in the legend.